trl/rewards/accuracy_rewards.py

```diff
@@ -79,10 +70,7 @@ def accuracy_reward(completions: list[list[dict[str, str]]], solution: list[str]
                 extraction_mode="first_match",
             )
             # Compute binary rewards if verifiable, `None` otherwise to skip this example
-            try:
-                reward = float(verify(gold_parsed, answer_parsed))
-            except Exception:
-                reward = None
```

Comment:
we don't need this try/except anymore, as both gold_parsed and answer_parsed are parsed